testdata/project-v2/Makefile

```diff
@@ -52,7 +52,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
Is this change related to enabling the goimports linter?

Reply:
To do the PR's we need to run `make generate` which will update the testdata.
Some PR was merged in the master without this update. (may the PR passed in the CI and was merged after the change which affects this artefact)

So, until it is merged (ps.: has a PR already for that  + many ones :-P ) you will see that in all diffs since whiteout it is not possible to check the CI. Mainly for we allow do the reviews.

Reply:
#1288 fixes it in a separate PR and is tracking all the affected PRs.